### PR TITLE
Added sandwichable support.

### PR DESCRIPTION
**Pull request by HappyMolly**
Hello, I added support for sandwichable.

### CODE REVIEW

**Comment by copygirl** [comment]
Thank you for the contribution, but I don't think this is right quite yet. You created a new tag in `sandwichable` namespace. If anything, it should be in `pocketbags` namespace. But creating a new tag is probably not the right way to go about it. So far, I think it's best to just include the items from that mod into `pouchable.json`. (If the mod is not present it looks like it ignores the entries.)

Also, the spacing is off. Some of those entries are spaced with tabs.

File: src/main/resources/data/pocketbags/tags/items/pouchable.json [hunk 1]
```diff
@@ -5,6 +5,15 @@
...
     "#c:seeds",
 
+    "sandwichable:cucumber",
+    "sandwichable:cucumber_seeds",
+    "sandwichable:lettuce_head",
+    "sandwichable:lettuce_seeds",
+    "sandwichable:onion",
+    "sandwichable:onion_seeds",
+    "sandwichable:tomato",
+    "sandwichable:tomato_seeds"
```
Comment (copygirl) [inline src/main/resources/data/pocketbags/tags/items/pouchable.json]: Missing comma.

File: src/main/resources/data/pocketbags/tags/items/pouchable.json [hunk 2]
```diff
@@ -5,6 +5,15 @@
     "#saplings",
     "#c:seeds",
 
+    "sandwichable:cucumber",
+    "sandwichable:cucumber_seeds",
```
Comment (copygirl) [inline src/main/resources/data/pocketbags/tags/items/pouchable.json]: On second thought, the seeds should go into `c/tags/items/seeds.json`.

File: src/main/resources/data/pocketbags/tags/items/pouchable.json [hunk 3]
```diff
@@ -5,6 +5,15 @@
     "#saplings",
     "#c:seeds",
 
+    "sandwichable:cucumber",
```
Comment (copygirl) [inline src/main/resources/data/pocketbags/tags/items/pouchable.json]: Modded items should probably go underneath `minecraft`.

**Review by copygirl — CHANGES_REQUESTED** [review]

**Comment by copygirl** [comment]
I hope it's okay to request this many changes. I could easily add the support myself exactly how I want it, but I figure you want to be the one to do it..?

**Comment by HappyMolly** [comment]
It's good practice for me, sorry for making so sloppy pull requests.

**Review by copygirl — APPROVED** [review]

**Pull request merged**